install_config/install/deploy_router.adoc

```diff
@@ -609,6 +609,58 @@ $ echo "    router metrics at: http://<ip>:9090/consoles/haproxy.html "
 ====
 
 
+=== Preventing Connection Failures During Restarts ===
```

Comment:
You can remove the concluding === in the heading (we only need the ones in the beginning)